Makefile

```diff
@@ -9,7 +9,7 @@ all: $(MKFS)
 	make -C $(KDIR) M=$(PWD) modules
 
 IMAGE ?= test.img
-IMAGESIZE ?= 50
+IMAGESIZE ?= 200
```

Comment:
Why did you change?

Reply:
You shall mention the motivation along with the formula in git commit message(s) as well.